scene/3d/camera_3d.h

```diff
@@ -88,6 +88,7 @@ class Camera3D : public Node3D {
 	friend class Viewport;
 	void _update_audio_listener_state();
 	TypedArray<Plane> _get_frustum() const;
+	TypedArray<Vector3> _get_frustum_plane_intersection(const Plane &p_plane) const;
```

Comment:
I really need to not do these half a sleep. Forgot PackedVector3 is a thing. Will swap tomorrow... uh today?

Reply:
Don't worry - take your time. `PackedVector3` is preferred over typed array (where it is possible) of course.